lib/iris/fileformats/pp.py

```diff
@@ -1286,8 +1286,8 @@ def data(self):
 
         """
         # Cache the real data on first use
-        if isinstance(self._data, biggus.Array):
-            data = self._data.masked_array()
+        if iris._lazy_data.is_lazy_data(self._data):
+            data = iris._lazy_data.as_concrete_data(self._data)
             if ma.count_masked(data) == 0:
```

Comment:
I don't think this line works any more, as the result as ```as_concrete_data``` doesn't always yield a masked array,